drv/sidecar-seq-api/src/lib.rs

```diff
@@ -44,4 +50,58 @@ pub enum TofinoSequencerPolicy {
     RestartOnFault = 2,
 }
 
+#[derive(
+    Copy, Clone, Debug, PartialEq, Eq, Deserialize, Serialize, SerializedSize,
+)]
+#[repr(C)]
```

Comment:
This `repr` seems unnecessary, since we're always serializing the value with `hubpack`